### PR TITLE
drivers/bmx055: fix crazy use of FPU

### PR DESCRIPTION
**Pull request by maribu**
### Contribution description

As the title says...

### Testing procedure

Code review should be sufficient

### Issues/PRs references

None

### CODE REVIEW

File: drivers/bmx055/bmx055.c [hunk 1]
```diff
@@ -234,7 +234,7 @@ int bmx055_gyro_read(const bmx055_t *dev, int16_t *data)
     uint16_t scale;
 
     /* converting scale info into real scaling values */
-    scale = (GYRO_2000_DPS / pow(2, dev->p.gyro_scale));
+    scale = (GYRO_2000_DPS / (1U << dev->p.gyro_scale));
```
Comment (benpicco) [inline drivers/bmx055/bmx055.c:237]: or as @chrysn pointed out 

```suggestion
    scale = GYRO_2000_DPS >> dev->p.gyro_scale;
```

**Comment by benpicco** [comment]
Please squash!

**Comment by maribu** [comment]
bors merge

**Comment by kaspar030** [comment]
> GitHub status checks took too long to complete, so bors is giving up. You can adjust bors configuration to have it wait longer if you like.

after 41min?

**Comment by maribu** [comment]
bors merge

**Pull request merged**